code/game/gamemodes/cult/items/tome.dm

```diff
@@ -8,6 +8,83 @@
 	w_class = 2.0
 	unique = TRUE
 	slot_flags = SLOT_BELT
+	var/obj/item/inserted_item  //chalk code graciously stolen from the PDA's pen code   -MalMalmulam
+	var/list/contained_item = list(/obj/item/pen/drafting/) 
+	var/bloody = ""
+
+/obj/item/pen/drafting/red/cult
+	name = "engraved chalk"
+	desc = "A piece of chalk for marking areas of floor, or for drawing.  This one has strange symbols engraved on it."
+	color = COLOR_HUMAN_BLOOD
+	colorName = "redc"	
+
+/obj/item/book/tome/Initialize(mapload, ...)
+	. = ..()
+	inserted_item =	new /obj/item/pen/drafting/red/cult(src)
+	var/list/blooddonors = list()  //the chalk turns the color of a random nearby cultist's blood, if there is one
+	for(var/mob/living/carbon/C in range(1, get_turf(src)))
+		if(!iscultist(C)) 
+			continue
+		blooddonors += C
+		if(blooddonors.len)
+			var/mob/living/carbon/blooddonor = pick(blooddonors)
+			inserted_item.color = blooddonor.species.blood_color
+
+
+obj/item/book/tome/proc/remove_chalk(mob/user)
+	if (!istype(user))
+		return
+
+	switch (use_check(user, USE_DISALLOW_SILICONS, show_messages = FALSE))
```

Comment:
This entire switch isn't necessary at all, just do
``if(use_check_and_message(user))
    return```

Reply:
Gotcha!  Every day I learn something new : D Man, I copied that straight from the PDA's code... that code must be OLD.

Reply:
Fixed.